command/agent/testdata/basic.hcl

```diff
@@ -105,6 +105,7 @@ client {
   cni_path              = "/tmp/cni_path"
   bridge_network_name   = "custom_bridge_name"
   bridge_network_subnet = "custom_bridge_subnet"
+  bridge_network_subnet_ipv6 = "custom_bridge_subnet_ipv6"
```

Comment:
nit: the `=`s should all be in alignment

Reply:
oh that might be what this test failure is complaining about: https://github.com/hashicorp/nomad/actions/runs/10255235476/job/28371795890?pr=23747#step:6:43

Reply:
Ohh thank you! I was looking for what part of the linting I was missing 👍